templates/systems/element_detail_tabs.html

```diff
@@ -410,4 +422,10 @@ <h3><button type="submit" id="new_component_statement" class="small btn btn-md b
         }
       </script>
 
+      <script>
+        $( document ).ready(function() {
```

Comment:
This is probably where edit will get its functionality

Reply:
Not sure what you mean by the comment.